### PR TITLE
DTV Viscountess

### PR DESCRIPTION
**Pull request by rf-l**
Added the Viscountess to spawn in DTV.

### CODE REVIEW

**Review by Muparadzi — CHANGES_REQUESTED** [review]
I need to modify the client messages to get viscount agent and display their agent name rather than hardcoded 'Viscount'.

**Review by Muparadzi — DISMISSED** [review]

File: src/Module.Server/Modes/Dtv/CrpgDtvClient.cs [hunk 1]
```diff
@@ -134,15 +137,16 @@ private void HandleViscountDeath(CrpgDtvGameEnd message)
     private void HandleViscountUnderAttack(CrpgDtvViscountUnderAttackMessage message)
     {
         var attackerAgent = Mission.MissionNetworkHelper.GetAgentFromIndex(message.AgentAttackerIndex, true);
+        var viscountAgent = Mission.MissionNetworkHelper.GetAgentFromIndex(message.AgentVictimIndex, true);
 
         if (attackerAgent == null)
```
Comment (namidaka) [inline src/Module.Server/Modes/Dtv/CrpgDtvClient.cs:142]: If we're going throught the hassle of making the code allow for any kind of agent to defend , then the code should be made "viscount agnostic" .
 The agent to defend can be called VIP or vip_to_defend.

**Review by namidaka — CHANGES_REQUESTED** [review]

File: src/Module.Server/Modes/Dtv/CrpgDtvClient.cs (hunk 1 above)
Comment (Muparadzi) [reply src/Module.Server/Modes/Dtv/CrpgDtvClient.cs]: I have refactored references to 'viscount' in the code to VIP.

The check for "crpg_spawn_viscount" tag remains the same until a coordinated effort can be made to update scenes.

Comment (Muparadzi) [reply src/Module.Server/Modes/Dtv/CrpgDtvClient.cs]: > The check for "crpg_spawn_viscount" tag remains the same until a coordinated effort can be made to update scenes.

Spawn tag has been updated to `crpg_spawn_vip` and @rf-l has updated scenes accordingly.

**Pull request merged**